### PR TITLE
Final fix lvsetup v2

### PR DESCRIPTION
**Pull request by hathyaga**

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@hathyaga please squash  commits in single commit

**Comment by hathyaga** [comment]
[lvsetup_fix_joblog.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/803370/lvsetup_fix_joblog.txt)

**Comment by hathyaga** [comment]
Commits squashed

File: io/disk/lvsetup.py [hunk 1]
```diff
@@ -45,11 +46,19 @@ def setUp(self):
...
         lv_name = self.params.get('lv_name', default='avocado_lv')
         self.lv_size = self.params.get('lv_size', default='1G')
-        self.fs_name = self.params.get('fs', default='ext4')
+        self.fs_name = self.params.get('fs', default='ext4').lower()
+        if self.fs_name == 'xfs':
+            pkg = 'xfsprogs'
+        if self.fs_name == 'btrfs':
+            pkg = 'btrfs-progs'
+        if not smm.check_installed(pkg) and not smm.install(pkg):
+            self.skip("%s package is need to test" % pkg)
```
Comment (apahim) [inline io/disk/lvsetup.py]: `s/need/needed`. But a better message would be: `"Required package '%s' is missing and could not be installed."`

**Comment by PraveenPenguin** [comment]
@hathyaga please rebase your commit with updated your code from  master   (upstream)

**Comment by hathyaga** [comment]
Closing this. NEW PR: https://github.com/avocado-framework-tests/avocado-misc-tests/pull/289

**Pull request closed without merging**